### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

File: lib/models.py [hunk 1]
```diff
@@ -225,7 +225,7 @@ def vote(self, dashd, signal, outcome):
             self.sync_network_vote(dashd, signal)
 
     def sync_network_vote(self, dashd, signal):
-        printdbg('\tsyncing network vote for object %s with signal %s' % (self.object_hash, signal.name))
+        printdbg('\tSyncing network vote for object %s with signal %s' % (self.object_hash, signal.name))
```
Comment (nmarley) [inline lib/models.py:228]: Not technically a typo, and this _could_ affect if anyone has created a tool to look for specific text in the logs... but honestly probably not an issue. Will wait for more feedback to see what others think.

File: lib/models.py [hunk 2]
```diff
@@ -701,7 +701,7 @@ def check_db_sane():
     for model in db_models():
         if not getattr(model, 'table_exists')():
             missing_table_models.append(model)
-            printdbg("[warning]: table for %s (%s) doesn't exist in DB." % (model, model._meta.db_table))
+            printdbg("[warning]: Table for %s (%s) doesn't exist in DB." % (model, model._meta.db_table))
```
Comment (nmarley) [inline lib/models.py:704]: Same as above

**Comment by nmarley** [comment]
Thanks for the PR! I have left a couple comments inline... mostly small/trivial, so not likely an issue.

**Review by UdjinM6 — APPROVED** [review]
utACK

Agree with @nmarley that fixed "typos" in `models.py` are not technically typos but I guess it won't hurt to fix them as well to follow common log style.

**Pull request merged**